radicle-cli/src/commands/patch/checkout.rs

```diff
@@ -0,0 +1,61 @@
+use crate::terminal as term;
+use radicle::cob::patch::{self, PatchId};
+use radicle::git;
+use radicle::profile::Profile;
+use radicle::storage::git::Repository;
+
+pub fn run(
+    storage: &Repository,
+    profile: &Profile,
+    git_workdir: &git::raw::Repository,
+    patch_id: &PatchId,
+) -> anyhow::Result<()> {
+    let patchs = patch::Patches::open(profile.public_key, storage)?;
+    let Some(patch) = patchs.get(patch_id)? else {
+        anyhow::bail!("Patch `{patch_id}` not found");
+    };
+
+    let spinner = term::spinner("Performing patch checkout...");
+
+    // Getting the patch obj!
+    let patch_haed = patch.head();
```

Comment:
```suggestion
    let patch_head = patch.head();
```

Or alternatively, just call it `head`